inc/budget.class.php

```diff
@@ -221,10 +232,10 @@ function getSearchOptions() {
       $tab[6]['name']            = __('End date');
       $tab[6]['datatype']        = 'date';
 
-      $tab[4]['table']           = $this->getTable();
```

Comment:
I think this change can cause a (minor) problem.
If a user has stored this searchoption in his preferences, the displayed column will changed
You need to keep index 4 for value field or provide an update line to change glpi_displaypreferences.